### PR TITLE
falco rule naming convention

### PR DESCRIPTION
**Pull request by Kaizhe**
Signed-off-by: kaizhe <derek0405@gmail.com>

**What type of PR is this?**

/kind documentation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area rules

/area proposals

**What this PR does / why we need it**:

Propose falco rule naming convention

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
proposal: rules naming convention
```

### CODE REVIEW

File: proposals/20191217-rules-naming-convention.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Propose some basic naming conventions when new lists, macros, rules are introduced. 
+
+## Motivation
+
+We want to help people from the community to contribute to falco rules. It will help improving the security content provided by Falco out of the box. Since people have different preference of naming things, it's necessary to set forth some basic naming convention for people to follow when creating new rules, macros and lists.
+
+### Goals
+
+People will have to follow the naming conventions rules when introducing new falco rules, macros and lists.
```
Comment (leodido) [inline proposals/20191217-rules-naming-convention.md]:
```suggestion
People will have to follow the naming conventions rules when introducing new Falco rules, macros and lists.
```

File: proposals/20191217-rules-naming-convention.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+### Non-Goals
+
+There will be no intention to cover Falco syntax in this proposal.
```
Comment (leodido) [inline proposals/20191217-rules-naming-convention.md]: Do you mean Falco rules syntax?

Comment (Kaizhe) [reply proposals/20191217-rules-naming-convention.md]: yes

Comment (Kaizhe) [reply proposals/20191217-rules-naming-convention.md]: will update

**Comment by leodido** [comment]
Ok Khaize almost there: sign-off all the commits :)

**Pull request merged**